### ISSUE TITLE
suggestions for calculate_best_node

### ISSUE THREAD

**Issue by shewenkan** [codelucas/newspaper#965]
the score of the grandpa node should not be discounted if it has son text_node with better score，for example: https://indianexpress.com/article/opinion/columns/c-raja-mohan-writes-cooperation-amid-conflict-is-indias-burden-for-g20-8472106/
the correct top_node is the div id="pcl-full-content" not div class="ev-meter-content"

**Comment by Dustin-MOP** [comment]
I agree 100 %. I am facing similar issues on multiple pages like this: 
https://care-vita.de/2021/11/15/pflegefachkraft-in-werder/

Here, only the portion between "Deine Aufgaben" and the list above "Dein Profil" is considered as the "text" of that page. However, obviously that's not the case.
Maybe consider incorporating the search for h1, h2, etc. and rather taking the "outer wrapper node" (one that holds multiple "good" nodes/wrapper nodes)

I am not very experienced with Github otherwise I'd prepare a code recommendation...
